model-optimizer/install_prerequisites/install_prerequisites.sh

```diff
@@ -83,18 +83,129 @@ elif [[ "$OSTYPE" == "darwin"* ]]; then
+        if check_ov_package "$python_executable" "$requires_sudo"; then
+            if $mo_is_custom; then
+                print_warning "Existing \"OpenVINO (TM) Toolkit\" pip version is incompatible with Model Optimizer."
+                print_warning "For not release Model Optimizer version please build Inference Engine Python API from sources (preferable) or install latest \"OpenVINO (TM) Toolkit\" version using pip install openvino (may be incompatible)."
+            else
+                print_warning "Existing \"OpenVINO (TM) Toolkit\" pip version is incompatible with Model Optimizer."
+                print_warning "For release Model Optimizer version which is $mo_release_version please install \"OpenVINO (TM) Toolkit\" using pip install openvino==$mo_release_version or build Inference Engine Python API from sources."
+            fi
+            return 0
+        fi
+
+        print_warning "No available Inference Engine Python API was found. Trying to install \"OpenVINO (TM) Toolkit\" using pip."
+
+        if $mo_is_custom; then
+            print_warning "Custom Model Optimizer version detected."
+            print_warning "The desired version of Inference Engine can be installed only for release Model Optimizer version."
+            print_warning "The latest \"OpenVINO (TM) Toolkit\" version will be installed (may be incompatible with current Model Optimizer version)."
+            print_warning "It is recommended to build Inference Engine from sources even if installation will be successful."
+        elif install_ov "$python_executable" "$requires_sudo" "$mo_release_version"; then
+            if check_ie "$python_executable"; then
+                return 0
+            fi
+
+            print_warning "Installed \"OpenVINO (TM) Toolkit\" version $mo_release_version doesn't work as expected...Uninstalling..."
+            uninstall_ov "$python_executable" "$requires_sudo"
+            print_warning "Please consider to build Inference Engine Python API from sources"
+            return 0
+        else
+            print_warning "Can not find \"OpenVINO (TM) Toolkit\" version $mo_release_version in pip."
+            print_warning "But the latest \"OpenVINO (TM) Toolkit\" version will be installed (may be incompatible with current Model Optimizer version)."
+            print_warning "It is recommended to build Inference Engine from sources even if installation will be successful."
+        fi
+
+        #install the latest OpenVINO pip version
+        if install_latest_ov "$python_executable" "$requires_sudo"; then
+            if check_ie "$python_executable"; then
+                return 0
+            else
+                print_warning "Installed latest \"OpenVINO (TM) Toolkit\" version doesn't work as expected...Uninstalling..."
+                uninstall_ov "$python_executable" "$requires_sudo"
+                print_warning "Please consider to build Inference Engine Python API from sources."
```

Comment:
```suggestion
                print_warning "Please consider building the Inference Engine Python API from sources."
```

Reply:
I would prefer to have the same message about OV installation in each place.
Earlier you suggested this: "Consider building the Inference Engine Python API from sources." Is it ok?